pulsar-client/src/main/java/org/apache/pulsar/client/impl/ClientCnx.java

```diff
@@ -107,7 +107,7 @@
     }
 
     public ClientCnx(ClientConfigurationData conf, EventLoopGroup eventLoopGroup) {
-        super(30, TimeUnit.SECONDS);
+        super(conf.getKeepAliveIntervalSeconds(), TimeUnit.SECONDS);
```

Comment:
This is changing the client side timeout to timeout a connection with a non-responsive broker. The broker timeout, though, is still set at 30seconds, so broker might still close the connection based on that timeout